src/css/style.css

```diff
@@ -58,13 +52,19 @@ main {
   /* transform: scale(0); */
 }
 
+.card img {
+  width: 200px;
+  height: 200px;
+}
+
 .abilities {
   display: flex;
   flex-wrap: wrap;
   gap: 20px;
 }
 
 .ability {
+  text-align: center;
   padding: 5px 20px;
   border-radius: 20px;
```

Comment:
- [OPTIONAL] You can use flexible units like `rem`, `em`, and percentages to achieve responsiveness easily.